resource-packs/Catppuccin UI/create_flavors.py

```diff
@@ -519,6 +522,11 @@ def main():
 def rgb_to_tuple(colorRGB):
     return (colorRGB.r, colorRGB.g, colorRGB.b)
 
+# Applies a -26% lightness adjustment to the color
+def darker_red(colorRGB):
+    hls_tuple = rgb_to_hls(*rgb_to_tuple(colorRGB))
+    red2_hls_tuple = (hls_tuple[0], hls_tuple[1] * 0.74, hls_tuple[2])
+    return tuple(round(i) for i in hls_to_rgb(*red2_hls_tuple))
```

Comment:
I didn't know how to do this more cleanly, would be open for anyone better to refactor this
The RGB values were pretty much +- 3 for all flavors